src/scss/_about.scss

```diff
@@ -211,6 +211,12 @@
 			}
 			img {
 			  margin: auto;
+			  display: block;
+			  height: 7rem;
```

Comment:
I specified a height here so that in tablet view everything is aligned when the addresses are rendered side-by-side.

<img width="646" alt="Screen Shot 2022-05-06 at 01 15 55" src="https://user-images.githubusercontent.com/405000/167041960-fae8ef64-6307-4b28-a3a9-4d4ea1d86395.png">